.github/workflows/packaging.yml

```diff
@@ -54,8 +54,12 @@ jobs:
           fi >> $GITHUB_ENV
       - name: Upload tarball Release Asset
         uses: softprops/action-gh-release@v2
-        if: startsWith(github.ref, 'refs/tags/') && github.repository == 'ROCm/omniperf'
+        if: startsWith(github.ref, 'refs/tags/') && github.repository == 'ROCm/rocprofiler-compute'
```

Comment:
Note that we'll have to rename the repository before merging your change, otherwise this check will fail.